### ISSUE TITLE
Inconsistant results

### ISSUE THREAD

**Issue by ahmdtaha** [alinlab/cs-kd#2]
The reported performance is 70% while the **best** performance is 66.741%.
Reporting the best is not even a good idea, because there is no validation set on CUB!

_The following is the experiment log_

==> Preparing dataset: CUB200
Number of train dataset:  5994
Number of validation dataset:  5794
==> Building model: densenet121
2
Using CUDA..
[2020-08-09 10:41:17,426] [main] /home/cs-kd-master/train.py
[2020-08-09 10:41:17,427] [main] Namespace(batch_size=32, cls=True, dataroot='~/data/', dataset='CUB200', decay=0.0001, epoch=200, lamda=3.0, lr=0.1, model='densenet121', name='2', ngpu=1, resume=False, saveroot='./results', sgpu=1, temp=4.0)

Epoch: 0
 [========================================= 188/188 
[2020-08-09 10:42:07,152] [train] [Epoch 0] [Loss 5.370] [cls 0.027] [Acc 0.634]
 [========================================= 182/182 
[2020-08-09 10:42:18,030] [val] [Epoch 0] [Loss 5.664] [Acc 1.294]
Saving..

Epoch: 1
 [========================================= 188/188 
[2020-08-09 10:43:05,969] [train] [Epoch 1] [Loss 5.196] [cls 0.034] [Acc 0.951]
 [========================================= 182/182 
[2020-08-09 10:43:16,492] [val] [Epoch 1] [Loss 5.197] [Acc 2.261]
Saving..

.................more epochs
Epoch: 198
 [========================================= 188/188 
[2020-08-09 13:55:04,013] [train] [Epoch 198] [Loss 0.617] [cls 0.277] [Acc 90.357]
 [========================================= 182/182 
[2020-08-09 13:55:15,010] [val] [Epoch 198] [Loss 1.500] [Acc 66.534]

Epoch: 199
 [========================================= 188/188 
[2020-08-09 13:56:01,925] [train] [Epoch 199] [Loss 0.647] [cls 0.280] [Acc 89.773]
 [========================================= 182/182 
[2020-08-09 13:56:13,293] [val] [Epoch 199] [Loss 1.499] [Acc 66.137]
Best Accuracy : 66.741455078125
[2020-08-09 13:56:13,293] [best] [Acc 66.741]

**Comment by sm3199** [comment]
For the densenet121, please try **'lamda=2'.**

In our reproduce, it achieves 69.31% as reported in the paper.

_The following is the experiment log_

... epoches
[2019-11-07 13:23:59,632] [train] [Epoch 198] [Loss 0.518] [KL 0.326] [Local 0.000] [Acc 91.175]
[2019-11-07 13:24:16,763] [val] [Epoch 198] [Loss 1.359] [Acc 68.485] [Top 5 88.281]
[2019-11-07 13:25:03,773] [train] [Epoch 199] [Loss 0.512] [KL 0.329] [Local 0.000] [Acc 91.575]
[2019-11-07 13:25:21,216] [val] [Epoch 199] [Loss 1.352] [Acc 68.640] [Top 5 88.989]
[2019-11-07 13:25:21,219] [best] [Acc 69.313] [Top 5 88.799]

**Comment by ahmdtaha** [comment]
Thanks for the update
